src/EditorFeatures/CSharp/StringCopyPaste/KnownSourcePasteProcessor.cs

```diff
@@ -83,24 +84,7 @@ private ImmutableArray<TextChange> GetEditsForNonRawString()
         {
             using var _ = PooledStringBuilder.GetInstance(out var builder);
 
-            if (StringExpressionBeforePaste is LiteralExpressionSyntax literal)
-            {
-                AppendContentForLiteral(builder);
-            }
-            else if (StringExpressionBeforePaste is InterpolatedStringExpressionSyntax interpolatedString)
-            {
-                AppendContentForInterpolatedString(builder);
-            }
-            else
-            {
-                throw ExceptionUtilities.Unreachable;
-            }
-
-            return ImmutableArray.Create(new TextChange(_selectionBeforePaste.Span.ToTextSpan(), builder.ToString()));
-        }
```

Comment:
this method called out to two methods that were almost identical in how they behaved.  so i merged those two methods into one, and then tweaked it in the one place they differ in behavior.